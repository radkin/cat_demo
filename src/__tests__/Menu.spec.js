import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import Menu, { Counter, dataReducer } from '../Hamburger/Menu';

describe('Menu', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<Menu />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
