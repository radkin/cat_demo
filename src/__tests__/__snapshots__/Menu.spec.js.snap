// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu snapshot renders 1`] = `
<div
  style={
    Object {
      "background": "#845286",
      "color": "#fafafa",
      "display": "flex",
      "flexDirection": "column",
      "height": 0,
      "left": 0,
      "opacity": 0.95,
      "position": "absolute",
      "top": 0,
      "transition": "height 0.3s ease",
      "width": "100vw",
      "zIndex": 2,
    }
  }
/>
`;
